packages/devextreme/testing/testcafe/tests/navigation/tabs/common.ts

```diff
@@ -110,6 +110,57 @@ test('Tabs with width: auto in flex container', async (t) => {
   return createWidget('dxTabs', { dataSource, width: 'auto' }, '#tabs');
 });
 
+test('Tabs wrapper width changing must update tab item width', async (t) => {
```

Comment:
Why do we need screenshots here?
I believe we can just check that width is correct.
Moreover, it can be done in QUnit because we don't emulate any user action here